behat.yml

```diff
@@ -0,0 +1,31 @@
+# Run mfa behat tests with this command
+# Note that mfa behat tests require CMS module
+# ========================================================================= #
+# chromedriver
+# vendor/bin/behat @mfa
+# ========================================================================= #
+default:
+  suites:
+    mfa:
+      paths:
+        - %paths.modules.mfa%/tests/Behat/features
+      contexts:
+        - SilverStripe\Framework\Tests\Behaviour\FeatureContext
+        - SilverStripe\Framework\Tests\Behaviour\CmsFormsContext
+        - SilverStripe\Framework\Tests\Behaviour\CmsUiContext
+        - SilverStripe\BehatExtension\Context\BasicContext
+        - SilverStripe\BehatExtension\Context\EmailContext
+        - SilverStripe\MFA\Tests\Behat\Context\LoginContext
```

Comment:
Do behat configs work in a similar way to core config merges? Will this be sufficient to override the core login context, or does omitting it from this list achieve the same thing?

Reply:
I'm not sure what you mean, but this context is overriding the core one in order to augment the behaviour of "I log in with X permission" - I've removed the core one from this list for that reason